The 'sh' completion file only completes on *.sh files.
This breaks the expectation (on Slackware, at least) that
e.g. "sh /etc/rc.d/rc.my<TAB>" will complete to rc.mysql.
This patch fixes that, but a consequence is that "sh" will
complete on *any* filename.  IMHO, that's not a regression
at all, since stock bash without bash-completion does that
already...  --rworkman

Acked-by: volkerdi

diff -Nur bash-completion-1.3.orig/completions/sh bash-completion-1.3/completions/sh
--- bash-completion-1.3.orig/completions/sh	2011-01-21 03:36:11.000000000 -0600
+++ bash-completion-1.3/completions/sh	2011-03-12 02:06:00.964938285 -0600
@@ -28,7 +28,7 @@
         return 0
     fi
 
-    _filedir sh
+    _filedir
 } &&
 complete -F _sh sh
 
